webapp/channels/src/components/tabs/tab/index.tsx

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import type {TabProps} from 'react-bootstrap';
+import {Tab as ReactBootstrapTab} from 'react-bootstrap';
+
+type Props = TabProps & {
+    children?: React.ReactNode;
+}
```

Comment:
Same as my comment in tabs. I would even remove the `tabClassName` prop for now. For all that we know, all tabs will look exactly the same as the tab drafts. If that is not the case, we can look into it later.

Reply:
Same as above, I strongly think we should not.

Reply:
Done.